posts/2023-04-04-23.0.0.3.adoc

```diff
@@ -0,0 +1,600 @@
+
+[[openapi]]
+|https://github.com/eclipse/microprofile-open-api/releases/tag/3.1[MicroProfile OpenAPI 3.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/mpOpenAPI-3.1.html[mpOpenAPI-3.1]
+
+[[jwt]]
+|https://github.com/eclipse/microprofile-jwt-auth/releases/tag/2.1[MicroProfile JWT Authentication 2.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/mpJwt-2.1.html[mpJwt-2.1]
+
+|https://github.com/eclipse/microprofile-config/releases/tag/3.0.2[MicroProfile Config 3.0]
+|Unchanged
+|link:{url-prefix}/docs/latest/reference/feature/mpConfig-3.0.html[mpConfig-3.0]
+
+|https://github.com/eclipse/microprofile-health/releases/tag/4.0.1[MicroProfile Health 4.0]
+|Unchanged
+|link:{url-prefix}/docs/latest/reference/feature/mpHealth-4.0.html[mpHealth-4.0]
+
+|https://github.com/eclipse/microprofile-rest-client/releases/tag/3.0.1[MicroProfile Rest Client 3.0]
+|Unchanged
+|link:{url-prefix}/docs/latest/reference/feature/mpRestClient-3.0.html[mpRestClient-3.0]
+
+|https://github.com/eclipse/microprofile-fault-tolerance/releases/tag/4.0.2[MicroProfile Fault Tolerance 4.0]
+|Unchanged
+|link:{url-prefix}/docs/latest/reference/feature/mpFaultTolerance-4.0.html[mpFaultTolerance-4.0]
+
+|https://jakarta.ee/specifications/coreprofile/10/[Jakarta EE Core Profile 10]
+
+|New
+|<<coreprofile, See Core Profile table>>
+
+
+|===
+
+You can enable the MicroProfile 6 feature by using the `microProfile-6.0` feature in the server.xml file:
+
+[source,xml]
+----
+<featureManager>
+    <feature>microProfile-6.0</feature>
+</featureManager>
+----
+    
+To find out more, take a look at the MicroProfile 6.0 https://download.eclipse.org/microprofile/microprofile-6.0/microprofile-spec-6.0.html[specification], https://github.com/eclipse/microprofile/releases/tag/6.0-RC3[release], and link:{url-prefix}/docs/latest/reference/javadoc/microprofile-6.0-javadoc.html[Javadoc]
+
+   
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/24759
+// Contact/Reviewer: gjwatts,ReeceNana
+// // // // // // // // 
+[#java20]
+== Support for Java SE 20
+
+Java 20 includes the following features and changes:
+
+* 429: link:https://openjdk.org/jeps/429[Scoped Values (Incubator)]
+* 432: link:https://openjdk.org/jeps/432[Record Patterns (Second Preview)]
+* 433: link:https://openjdk.org/jeps/433[Pattern Matching for switch (Fourth Preview)]
+* 434: link:https://openjdk.org/jeps/434[Foreign Function & Memory API (Second Preview)]
+* 436: link:https://openjdk.org/jeps/436[Virtual Threads (Second Preview)]
+* 437: link:https://openjdk.org/jeps/437[Structured Concurrency (Second Incubator)]
+
+
+To use Java 20: 
+
+1. link:https://jdk.java.net/20/[Download the early access release of Java 20]. 
+2. Install the <<run,23.0.0.3-beta>> version of Open Liberty.

```

Comment:
Please change
```
Install the <<run,23.0.0.3-beta>> version of Open Liberty.
```
to
```
Install the <<run,23.0.0.3>> version of Open Liberty.
```

Reply:
Well install is a good word if you are using an Open Liberty zip which you downloaded from openliberty.io ... but I think you are saying that the wording should reflect the fact that some people may be trying this out through maven (I don't think we have a Java 20 docker image yet?).

Reply:
```suggestion
2. Obtainl the <<run,23.0.0.3>> version of Open Liberty.

```